aten/src/ATen/CMakeLists.txt

```diff
@@ -161,31 +161,31 @@ endif(MKLDNN_FOUND)
 
 list(APPEND ATen_CPU_DEPENDENCY_LIBS cpuinfo)
 
-if(NOT MSVC)
-  # Preserve values for the main build
-  set(__aten_sleef_build_shared_libs ${BUILD_SHARED_LIBS})
-  set(__aten_sleef_build_tests ${BUILD_TESTS})
-
-  # Unset our restrictive C++ flags here and reset them later.
-  # Remove this once we use proper target_compile_options.
-  set(OLD_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
-  set(CMAKE_CXX_FLAGS)
-
-  set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build sleef static" FORCE)
-  set(BUILD_DFT OFF CACHE BOOL "Don't build sleef DFT lib" FORCE)
-  set(BUILD_GNUABI_LIBS OFF CACHE BOOL "Don't build sleef gnuabi libs" FORCE)
-  set(BUILD_TESTS OFF CACHE BOOL "Don't build sleef tests" FORCE)
-  add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/sleef" ${CMAKE_BINARY_DIR}/sleef)
-  set_property(TARGET sleef PROPERTY FOLDER "dependencies")
-  list(APPEND ATen_THIRD_PARTY_INCLUDE ${CMAKE_BINARY_DIR}/include)
-  link_directories(${CMAKE_BINARY_DIR}/sleef/lib)
-  list(APPEND ATen_CPU_DEPENDENCY_LIBS sleef)
-
-  set(CMAKE_CXX_FLAGS ${OLD_CMAKE_CXX_FLAGS})
-
-  # Set these back. TODO: Use SLEEF_ to pass these instead
-  set(BUILD_SHARED_LIBS ${__aten_sleef_build_shared_libs} CACHE BOOL "Build shared libs" FORCE)
-  set(BUILD_TESTS ${__aten_sleef_build_tests} CACHE BOOL "Build tests" FORCE)
+if(NOT MSVC AND NOT EMSCRIPTEN)
+ # Preserve values for the main build
+ set(__aten_sleef_build_shared_libs ${BUILD_SHARED_LIBS})
+ set(__aten_sleef_build_tests ${BUILD_TESTS})
+
+ # Unset our restrictive C++ flags here and reset them later.
+ # Remove this once we use proper target_compile_options.
+ set(OLD_CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+ set(CMAKE_CXX_FLAGS)
+
+ set(BUILD_SHARED_LIBS OFF CACHE BOOL "Build sleef static" FORCE)
+ set(BUILD_DFT OFF CACHE BOOL "Don't build sleef DFT lib" FORCE)
+ set(BUILD_GNUABI_LIBS OFF CACHE BOOL "Don't build sleef gnuabi libs" FORCE)
+ set(BUILD_TESTS OFF CACHE BOOL "Don't build sleef tests" FORCE)
+ add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/sleef" ${CMAKE_BINARY_DIR}/sleef)
+ set_property(TARGET sleef PROPERTY FOLDER "dependencies")
+ list(APPEND ATen_THIRD_PARTY_INCLUDE ${CMAKE_BINARY_DIR}/include)
+ link_directories(${CMAKE_BINARY_DIR}/sleef/lib)
+ list(APPEND ATen_CPU_DEPENDENCY_LIBS sleef)
+
+ set(CMAKE_CXX_FLAGS ${OLD_CMAKE_CXX_FLAGS})
+
+ # Set these back. TODO: Use SLEEF_ to pass these instead
+ set(BUILD_SHARED_LIBS ${__aten_sleef_build_shared_libs} CACHE BOOL "Build shared libs" FORCE)
+ set(BUILD_TESTS ${__aten_sleef_build_tests} CACHE BOOL "Build tests" FORCE)
```

Comment:
Can we please avoid changing indentation unnecessarily?

Reply:
No idea why that happened actually, I must have had it commented out at some point!

Reply:
although, I just noticed everything else in that cmake file is indented.